backend/controller/user.go

```diff
@@ -62,6 +67,12 @@ func UpdateUser(w http.ResponseWriter, r *http.Request) {
 		}
 		user.Roles = roles
 	}
+	if reqUser.Email != "" {
```

Comment:
this is just not a put request anymore?

Reply:
we're not storing data such as the webauthnid on the frontend so i can't send all the user fields easily

Reply:
also this is fine for now but if we genuinely want a patch req here, I want to make a note that I'm gonna rewrite this logic

Reply:
cuz this is ugly asf